src/backend/expungeservice/models/charge_types/schedule_1_p_c_s.py

```diff
@@ -10,7 +10,11 @@ class Schedule1PCS(Charge):
 
     def _type_eligibility(self):
         if self.dismissed():
-            # TODO: Are violation dismissals for a PCS actually eligible? This does not appear to be the case.
+            if "violation" in self.level.lower():
+                return TypeEligibility(
+                    EligibilityStatus.INELIGIBLE,
+                    reason="Dismissed violations are ineligible by omission from statute"
+                )
             return TypeEligibility(EligibilityStatus.ELIGIBLE, reason="Dismissals are eligible under 137.225(1)(b)")
```

Comment:
I suggest wrapping this line in an `else` block since that's the style we use generally.